plugins/org.locationtech.udig.catalog.jgrass.tests/test-data/jgrass-test/newmapset/hist/ArcGrid

```diff
@@ -0,0 +1,8 @@
+Sat Oct 02 16:38:04 CEST 2021
+ArcGrid
+C:\dev\data\geodata\testdata\jgrass-db\jgrass-test\newmapset
+fgdrf
+DCELL
+
+
+Created by imageio-ext enabled JGrass
```

Comment:
file content is empty on master ;) please merge again

Reply:
Hmm, looks like auto-merge failed, just remove the lines manually or replace file with latest from master. :)